scripts/orchestrators/iac.bicep.sh

```diff
@@ -210,13 +210,25 @@ export -f deploy
 destroy() {
     local environmentName=${1}
     local layerName=${2}
+    local location=${3}
+    
+    deployments=$(az deployment sub list --query "[?tag=='GeneratedBy=symphony']" --query "[?tag=='EnvironmentName=${environmentName}']" --query "[?tag=='LayerName=${layerName}']" --query "[?location=='${location}']" --output json |jq -c -r '.[].name')
+    exit_code=$?
+    
+    for deployment in ${deployments}; do
+        echo "Deleting deployment : ${deployment}"
+        az deployment sub delete --name "${deployment}"
+        exit_code=$?
```

Comment:
is there a need for this variable? we override it in the loop, and later in the code.